docs/igm-roles.md

```diff
@@ -0,0 +1,39 @@
+# Informal Governance Model Definition & Roles
+
+_**Dependencies:** None_
+
+This doc defines what Quorum1's Informal Governance Processes is, how it relates to our Formal Governance Process, and what key roles are involved.
+
+## Note on Informal Governance Model
+
+Quorum1 is currently operating in our "Informal Governance Model" (IGM). 
+
+There is a deadline to adopt our "Formal Governance Model" (FGM) by December 31, 2025. The FGM will be a major release building on the ideation and design work which has been happening since before Quorum1's founding. FGM will be highly participatory, highly stigmergic, and designed to facilitate the continuing innovation and evolution of Quorum1 well into the future.
```

Comment:
Can we please reconsider the acronym "FGM"?

Reply:
We've already got legal docs which use that acronym... but yes let's tweak this in the next release.